src/NexusMods.App.UI/Pages/CollectionDownload/CollectionDownloadViewModel.cs

```diff
@@ -92,7 +92,7 @@ public CollectionDownloadViewModel(
             configureAwait: false
         );
 
-        CommandDownloadOptionalItems = _canDownloadOptionalItems.ToReactiveCommand<Unit>(
+        CommandDownloadOptionalItems = IsDownloading.CombineLatest(_canDownloadOptionalItems, static (isDownloading, canDownload) => !isDownloading && canDownload).ToReactiveCommand<Unit>(
```

Comment:
You can't download optional items while you are downloading the main ones?
Should we have two separate `IsDownloading` ? `IsDownloadingRequired` and `IsDownloadingOptional`?